models/comments.model.js

```diff
@@ -0,0 +1,22 @@
+const db = require("../db/connection")
+
+exports.removeCommentById = (comment_id) => {
+    let isNum = /^\d+$/.test(comment_id)
+    if (isNum === false) {
+        return Promise.reject({ status: 400, msg: "Invalid request"})
+    }
+    return db
+      .query(`SELECT comment_id FROM comments WHERE comment_id=$1`, [comment_id])
+      .then((result) => {
+        if (result.rows.length === 0) {
+          return Promise.reject({ status: 404, msg: "Comment not found" });
+        }
+      })
+      .then(() => {
+        return db.query(`DELETE FROM comments WHERE comment_id=$1`, [comment_id]);
```

Comment:
if you RETURNING * here you can check the length of the rows to send the 404 and not have to make the first db.query above (the SELECT one) 👆

but this way works fine too!